include/index_build_params.h

```diff
@@ -8,6 +8,7 @@ struct IndexFilterParams
   public:
     std::string save_path_prefix;
     std::string label_file;
+    std::string tags_file;
```

Comment:
start member var with underscore?

Reply:
May be you did not notice. These are public params and in code we make private variables with underscore.